server/store/sqlstore/store.go

```diff
@@ -50,6 +50,10 @@ func New(db *sql.DB, api plugin.API, enabledTeams func() []string, encryptionKey
 	}
 }
 
+func (s *SQLStore) Shutdown() error {
+	return s.db.Close()
+}
```

Comment:
cc @agnivade to add some perspective here -- I was hoping that the server would essentially "do this for free" when we shut down.

Reply:
Thanks, this is a good point. On the server side, we don't do the tracking of RPC connections at a plugin level. It will require some work, but not impossible. Then we can easily connect the plugin shutdown event to effectively closing those connections.